bin/mpm.json

```diff
@@ -0,0 +1,30 @@
+{
+  "title": "Example JSON Input for MPM",
+  "input_directory" : "/inputFiles/", 
+  "output_directory" : "/results/",
```

Comment:
Use a `post_processing` object. Include `output_directory` and `output_steps`. Change `/results` to `./results`